    private void foo1() throws RuntimeException, SQLException, OneMoreException {

    }

